### DISCUSSION TITLE
example of a grafana-server systemd service in case of the standalone binary download

### DISCUSSION THREAD

**Discussion by lite31** [grafana/grafana#45645] [Platform]
Hi,

Would you be so kind to point me to a systemd service file in case of a `grafana-server` **standalone binary** ?

(the one over here https://github.com/grafana/grafana/tree/main/packaging/rpm is not representative in case of of standalone binaries)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
